packages/veritone-react-common/src/components/FolderTree/Modals/EditFolder/index.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import cx from 'classnames';
+import { get, isNil } from 'lodash';
+import { shape, bool, func } from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import styles from './styles.scss';
+
+export default function ModifyFolder({
```

Comment:
Can you clarify in which scenario a user will open the `ModifyFolder` modal vs. the `EditFolder`modal.  Seems like there's some duplicate logic there, but if it's different enough that should be fine.  This component should also be rename to `EditFolder` right?

Reply:
My first idea is breaking the CRUD into 4 components: Create, Edit, Move, Delete and they are not related to each other. Then after integrating into Collection and Folder Tree must be supported to multiple select folders, I made the Move Folder Modal support to create a new folder with selecting the parent folder. That's leading to the problem. I've removed the old edit modal and change the modify modal's name. thank you for reviewing.